src/main/java/de/retest/recheck/printer/PrinterValueProvider.java

```diff
@@ -0,0 +1,15 @@
+package de.retest.recheck.printer;
+
+import java.io.Serializable;
+
+import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+
+public interface PrinterValueProvider {
+
+	boolean isDefault( String key, Serializable value );
+
+	static PrinterValueProvider of( DefaultValueFinder finder, IdentifyingAttributes attributes ) {
+		return (key, value) -> finder != null && finder.isDefaultValue( attributes, key, value );
```

Comment:
Why should we allow `finder` to be `null` (same for `ElementDifferencePrinter`)?

Reply:
The `DefaultValueFinder` is passed through the printers and retrieved from [RecheckImpl#usedFinders](https://github.com/retest/recheck/pull/53/files/a65594d5e454d0bbb2c9cba987fc8849bfa0bf2c#diff-8e60368308fb84436b2ad86ded4b895fR40). If someone does not pass an instance back and instead returns `null` in the `RecheckAdapter#getDefaultValueFinder` or for some reason the `currentStep` is not present in `RecheckImpl#usedFinders`, we get `null` for finder.
We could say for the `RecheckAdapter` that it should not return `null`. But for the second part, we need to adjust [RecheckImpl#printer](https://github.com/retest/recheck/pull/53/files/a65594d5e454d0bbb2c9cba987fc8849bfa0bf2c#diff-8e60368308fb84436b2ad86ded4b895fR41) to use a default value.

Reply:
If we look at the usage of `RecheckAdapter#getDefaultValueFinder()`, we can see that it is used, e.g., within `AttributesDifferenceFinder` without a `null` check. I think we can assume that it is part of the `RecheckAdapter` contract that the corresponding method must not return `null`.

How would you suggest to adapt second part?